lib/sdf-server/src/service/module/install_module.rs

```diff
@@ -147,7 +144,7 @@ pub async fn install_module(
             }
             variants.push(front_end_variant);
         } else {
-            return Err(ModuleError::SchemaNotFoundFromInstall(id));
+            return Err(ModuleError::NoSchemaVariantsInModule(id));
```

Comment:
Changed this type because it's used for two different purposes (one for schemas and one for modules).

Reply:
Great job! That would've been confusing to get otherwise